micrometer1-shim/src/main/java/io/opentelemetry/micrometershim/OpenTelemetryCounter.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.micrometershim;
+
+import static io.opentelemetry.micrometershim.Bridging.baseUnit;
+import static io.opentelemetry.micrometershim.Bridging.name;
+import static io.opentelemetry.micrometershim.Bridging.tagsAsAttributes;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.Measurement;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.instrument.util.MeterEquivalence;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.DoubleCounter;
+import io.opentelemetry.api.metrics.Meter;
+import java.util.Collections;
+import javax.annotation.Nullable;
+
+@SuppressWarnings("HashCodeToString")
```

Comment:
what is this warning about?

Reply:
I think since hashcode is defined but toString isn't. It seems like it'd be worth defining toString but will follow up